test/converters/xcresult3/converter.go

```diff
@@ -34,7 +34,7 @@ func majorVersion(document serialized.Object) (int, error) {
 	if err != nil {
 		return -1, err
 	}
-	return int(major.(uint64)), nil
+	return int(major.(uint64)), err
```

Comment:
nit: I would leave it as `nil` to better articulate that at this point there should be no error. `err` would be nil as well here so technically is correct. But for me the `nil` one reads better.

Reply:
Sorry, this is a leftover change from previous commits. This is the line that the old `errcheck` linter was complaining about (before merging #205), and I tried this change to see if this is the issue. It was still complaining, and now I reverted the change